app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -127,12 +127,14 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		/**
 		 * Shortcut used for changing the audio track
 		 */
-		var shortcutAudioTrack = Preference.int("shortcut_audio_track", KeyEvent.KEYCODE_MEDIA_AUDIO_TRACK)
+		var shortcutAudioTrack =
```

Comment:
> Please don't format this file as the formatting was perfectly fine.

Reply:
I've turned off auto-formatting as it's causing more hassle than good :+1: